pkg/queryfrontend/roundtrip.go

```diff
@@ -27,6 +27,8 @@ const (
 	seriesOp       = "series"
 )
 
+var opPattern = regexp.MustCompile("/api/v1/label/.+/values$")
```

Comment:
What about renaming `opPattern` to `labelValuesPattern`? In case we have more patterns to match in the future.

Reply:
renamed.